src/core/evaluator.js

```diff
@@ -479,7 +479,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
       // Creates native image decoder only if a JPEG image or mask is present.
       var nativeImageDecoder = null;
-      if (useNativeImageDecoder &&
+      if (useNativeImageDecoder === 'decode' &&
```

Comment:
With the comments above taken into account, this line should be:
`if (nativeImageDecoderSupport === NativeImageDecoding.DECODE &&`